docs/topic-guides/sessions/session-basics.rst

```diff
@@ -67,22 +67,59 @@ your own! Please refer to the :ref:`templating <templates>` documentation.
+  "Start" button.
+
+* Click on the "Start with options" link in
+  the "Start/Connect" dropdown menu available on the top right of the project
+  page header. Or on the "New session" button on the project sessions tab.
+  This action will bring you to a
+  lengthy :ref:`"Start Session" <start_with_options_anchor>` page with many
+  more   options to personalize your session before starting.
+
+* Use an :ref:`"Autostart Session" <create_autostart_links_anchor>` link.
+  The user experience is very similar to the quick start button. You can easily
+  :ref:`generate and distribute links <create_autostart_links_anchor>` starting
+  on the latest version of any branch or even embed specific references and
+  environmental variables to tailor the session to specific needs.
+
+
+.. _start_with_options_anchor:
+
+Start with options
+~~~~~~~~~~~~~~~~~~
+
+When starting a new session through the "Start with options" button from the
+project page header or the "New session" button on the project sessions tab,
+you will get a complete overview of the session you will start.
+One of the typical use cases for using the extended options is picking a
+non-default branch.
+
+For most of the other options, the default should work well. If you encounter problems
+with an environment (E.G. not enough space or crashing when starting), consider adding
+processing power, memory, or disk space.
+
+Here's the rundown of the configuration options.
 
 +------------------------------+-------------------------------------------------------------------------------------------+
 | Option                       | Description                                                                               |
 +==============================+===========================================================================================+
-| Branch                       | Default is ``master``. You can switch if you are working on another branch                |
+| Branch                       | Default is ``master`` or ``main``. You can switch if you are working on another branch    |
 +------------------------------+-------------------------------------------------------------------------------------------+
-| Commit                       | Default is the latest, but you can launch the environment from an earlier commit. This is |
-|                              | especially useful if your latest commit's build failed (see below) or you have unsaved    |
-|                              | work that was automatically recovered.                                                    |
+| Commit                       | Default is the running session (if any) or autosave commit (if any) or latest.            |
+|                              | You can pick a different one, and that's especially useful if your latest commit's build  |
+|                              | failed (see below) or you have unsaved                                                    | 
```

Comment:
```suggestion
|                              | Selecting a different commit can be useful to see the state of the project at a specific point in time, to avoid a commit in which the image does not build, or to ignore autosaved work. |
```